force-app/main/default/lwc/nksEmbeddedVideoPlayer/nksEmbeddedVideoPlayer.html

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div style="position: relative">
+        <div if:true={error}>
+            <div class="decorator-wrapper alert-positioning">
+                <div style="align-items: inherit" class="navds-alert navds-alert--error navds-alert--medium">
```

Comment:
Fungerer navds styling her? Dette er vel embedding komponenten, så trenger den at siden den blir embeddet på har den stylingen fra før av? Og hvis så kan du garantere det? Jeg vet at alle nav.no sider skal ha det, men hva med sharepoint etc?

Reply:
Embedding funker ikke i Sharepoint 😄

Reply:
Men vi burde nok sjekke at embedding fortsatt fungerer på sider uten navds stylesheetet 😄, selv om det ikke er sharepoint. Greit å vite om det er et "krav" for sider som vil embedde videoene våre.

Reply:
Stylingen fungerer ikke for sider som ikke er på nav.no selvsagt, da er det kun teksten som fremkommer. Det er likt som det var i aura-komponenten. Selve embedding fungerer i seg selv, fungerer jo på github-siden min f. eks.